### ISSUE TITLE
Devportal API overview page accessibility issues

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8370]
### Describe your problem(s)

1. Side menu links has no accessible name, innerText, or [title].

![image](https://user-images.githubusercontent.com/20179540/84795224-d2c23980-b014-11ea-8c21-61be7213b5b5.png)

2. expand has no keyboard access

![image](https://user-images.githubusercontent.com/20179540/84795636-63007e80-b015-11ea-9a84-7a40ce868448.png)

3. Show more link should have `aria-labelledby` reference to the title

![image](https://user-images.githubusercontent.com/20179540/84795876-a78c1a00-b015-11ea-8f8f-91dc87eee6a2.png)

4. Delete chip does not meet minimum contrast ratio

![image](https://user-images.githubusercontent.com/20179540/84796952-dfe02800-b016-11ea-9c50-1f5602b44b1f.png)

![image](https://user-images.githubusercontent.com/20179540/84797156-20d83c80-b017-11ea-808f-5567bad42539.png)

### How will you implement it

---
